giraffe/src/utils/PlotEnv.ts

```diff
@@ -104,11 +103,18 @@ export class PlotEnv {
     )
   }
 
-  public get yTicks(): number[] {
+  public get yTicks(): number[] | string[] {
+    const currLayerSpec = this.config.layers.map((_, i) => this.getSpec(i))[0]
+    if (
+      currLayerSpec &&
+      currLayerSpec.type === 'mosaic' &&
+      Array.isArray(currLayerSpec.yTicks)
+    ) {
+      return currLayerSpec.yTicks
```

Comment:
hrrm.. why are other yTicks directly on config and ours is on currLayerSpec? just confused.